web/src/app/admin/standard-answer/page.tsx

```diff
@@ -295,14 +295,25 @@ const StandardAnswersTable = ({
           </div>
         )}
         {paginatedStandardAnswers.length > 0 && (
-          <div className="mt-4 flex justify-center">
-            <PageSelector
-              currentPage={currentPage}
-              totalPages={totalPages}
-              onPageChange={handlePageChange}
-              shouldScroll={true}
-            />
-          </div>
+          <>
+            <div className="mt-4">
+              <Text>
+                Ensure that you have added the category to the relevant{" "}
+                <a className="text-blue-500" href="/admin/bot">
```

Comment:
Nit: prefer using colors from the [tailwind theme](https://github.com/danswer-ai/danswer/blob/main/web/tailwind-themes/tailwind.config.js) (in this case seems like `text-link`)